docs/feature-flags/creating-feature-flags/auto-operation/rollout.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Progressive rollout
+# sidebar_position:
+slug: /feature-flags/creating-feature-flags/auto-operation/progressive-rollout
+description: 'Check how the progressive rollout features work in Bucketeer'
+tags: ['guide', 'automation', 'rollout', 'progressive rollout']
+---
+
+import CenteredImg from '@site/src/components/centered-img/CenteredImg';
+
+The Progressive Rollout solution automates the release process of new features to your users by gradually providing the flag variation you choose as the [default strategy](/feature-flags/creating-feature-flags/targeting) to a larger percentage of your users. This approach helps you detect problems early on and reduce the impact.
+
+## How rollout works
+
+The flag must have **two variations** to use the rollout feature. For example, consider that you have a boolean flag with two variations: `true` and `false`. This flag determines whether new content is available to users or not. 
+
+Currently, the `false` variation is the [OFF variation](/feature-flags/creating-feature-flags/targeting#the-targeting-page), meaning users can't see the new content. However, you want to make the content available to all users within ten days. In this case, you can use the rollout feature to progressively roll out the `true` variation to 10% of your users daily. 
+
+On the first day of the rollout, 10% of your users will see the new content. On the second day, 20% will see it, and so on. After ten days, the rollout will be complete, and all users will have access to the new content. The image below summarizes the rollout process described.
+
+<CenteredImg
+  imgURL="img/feature-flags/auto-ops-rules/rollout.png"
+  alt="Rollout example"
+  wSize="550px"
+  borderWidth="1px"
+/>
+
+## Creating a rollout
+
+To add a rollout to a feature flag, you need to follow the steps below:
+
+1. Go to the **Auto Operation** tab on the feature Flag page details.
+2. Click the **+ Add** button.
+3. Choose the **Enable** operation and select **Progressive Rollout**.
+4. Configure the rollout and submit.
+
+:::caution Number of variations
+
+The progressive rollout only works for flags with **two variations**. 
```

Comment:
The progressive rollout only works for flags with **two variations** and **no running experiment**.